netflix-clone-yt-master/src/app/pages/login/login.component.ts

```diff
@@ -0,0 +1,43 @@
+declare var google: any;
+import { Component, OnInit, inject } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-login',
+  standalone: true,
+  imports: [CommonModule],
+  templateUrl: './login.component.html',
+  styleUrls: ['./login.component.scss']
+})
+export class LoginComponent implements OnInit {
+  private router = inject(Router);
+  ngOnInit(): void {
+    google.accounts.id.initialize({
+      client_id: '531121168764-fj0qmf042utb4q601id0dt2ojmdpe6ej.apps.googleusercontent.com',
+      callback: (resp: any)=> this.handleLogin(resp)
+    });
+
+    google.accounts.id.renderButton(document.getElementById("google-btn"), {
+      theme: 'filled_blue',
+      size: 'large',
+      shape: 'rectangle',
+      width: 350
+    })
+  }
+
+  private decodeToken(token: string){
```

Comment:
Consider adding a return type to this method for better type safety.

Reply:
ok👍